root/static/styles/forms.less

```diff
@@ -93,6 +93,8 @@ div.form-row-add {
     margin: (@form-margin / 2) 0;
 }
 
+div.form-row-add.short { width: @form-input-width-short + 2; }
```

Comment:
What does the `+ 2` do? (It actually looks better if I remove that, since the buttons align perfectly.)

Reply:
I just followed the +2 from form-row-add, but I'm fine with removing that if you feel it looks better :)

Reply:
It looks better in FF at least.  In Chrome I get the same as your screenshot, but in FF if I remove the `+ 2` I see this, which looks perfect:
![image](https://user-images.githubusercontent.com/1056556/196253767-43aa355f-fc1c-4135-8cde-0eb13a5c09f9.png)
Dunno why the text input is 202px in FF and 158px in Chrome.

Reply:
Me neither tbh. Wonder if we could port this to Autocomplete2 and get it to be consistent that way? :)

Reply:
Pushed a commit for that.  If you'd prefer we can merge this first and I can move it into a separate PR.